### ISSUE TITLE
Exception: Only binarized weights supported when use deploy_bw.prototxt in FINN

### ISSUE THREAD

**Issue by saraballeri** [zhaoweicai/hwgq#4]
Hi,
I would to generate an hardware design through FINN, passing it a prototxt of any BNN.
I don't understand if the folder [https://github.com/zhaoweicai/hwgq/tree/master/examples/imagenet](url) already contains prototxts suitable for FINN (as well as caffemodels in [https://github.com/zhaoweicai/hwgq#models](url)). 
As the guide of FINN says, I have tried to run:
`python FINN/bin/finn --device=pynqz1 --prototxt=FINN/inputs/deploy_bw.prototxt --mode=estimate`
in which `deploy_bw.prototxt` is the prototxt contained in [https://github.com/zhaoweicai/hwgq/tree/master/examples/imagenet/alex-hwgq-3ne-clip-poly-320k](url)
but i get the exception:
`....`
  `File "/home/user/FINN/FINN/backend/fpga/backend_fpga.py", line 64, in passConvertToFPGALayers
    ret += [layers_fpga.FPGABipolarConvThresholdLayer(L)]
  File "/home/user/FINN/FINN/backend/fpga/layers_fpga.py", line 337, in __init__
    raise Exception("Only binarized weights supported")
Exception: Only binarized weights supported`

Can you clarify me this doubt?
Thanks,
Sara

**Comment by maltanar** [comment]
Hi Sara,

As you can see from the error output this is not a HWGQ issue but a FINN issue, so this issue belongs to the [FINN GitHub repo](https://github.com/Xilinx/FINN/).

**Comment by saraballeri** [comment]
Yes, i saw there. But then all your prototxts and caffemodels are binarized? 1 bit weight and 1 bit activation?
Thanks,
Sara
